README.md

```diff
@@ -14,6 +14,8 @@ The KeyringController has three main responsibilities:
 
 `yarn add eth-keyring-controller`
 
+This library uses the Node.js `events` API. If you are using this library outside of a Node.js context, ensure that you have a polyfill for the `events` API (this is built-in to bundlers such as `webpack` and `browserify`).
```

Comment:
Webpack removed polyfills in v5: https://webpack.js.org/blog/2020-10-10-webpack-5-release/#automatic-nodejs-polyfills-removed. Maybe we want to just mention Browserify, for us?

``` suggestion
This library uses the Node.js `events` API. If you are using this library outside of a Node.js context, ensure that you have a polyfill for the `events` API (this is built-in to `browserify`).
```

Reply:
Good catch